src/elements/list-box.svelte

```diff
@@ -36,25 +38,48 @@ const LEFT = 'left';
 const RIGHT = 'right';
 
 let isDisabled: boolean;
+// needs to be calculated on initial render
```

Comment:
Timing thing. It just doesn't matter as much for `isDisabled` because it gets set pretty soon after. The problem with `displaySuffix` is that it's needed for the `let options = {...}` calculation below, which isn't set by the time the `generateOptions` function runs, so is always `undefined` because the `$` block that sets it will only run after the initial render, which is too late